src/web/app/pages-help/instructor-help-page/instructor-help-courses-section/instructor-help-courses-section.component.html

```diff
@@ -1,514 +1,356 @@
 <div>
-  <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes(true)">Courses</h2>
+  <h2 class="color-blue" *ngIf="!key || showQuestion.length > 0">Courses</h2>
   <!-- Section -->
-  <h4 *ngIf="displaySubsection(showQuestion, 0, 5)">Enrolling Students in Courses</h4>
+  <h4 *ngIf="displaySubsection(0, 5)">Enrolling Students in Courses</h4>
   <!-- Question -->
-  <div class="instr-help-qn-first" *ngIf="showQuestion.length === 0 || showQuestion[0]" #question id="{{CoursesSectionQuestions.COURSE_ADD_STUDENTS}}">
-    <p>
-      <button type="button" class="btn btn-outline-primary" (click)="questionsToCollapsed[CoursesSectionQuestions.COURSE_ADD_STUDENTS]= !questionsToCollapsed[CoursesSectionQuestions.COURSE_ADD_STUDENTS]" [attr.aria-expanded]="!questionsToCollapsed[CoursesSectionQuestions.COURSE_ADD_STUDENTS]">
-      How do I add students to a course?
-      </button>
-    </p>
-    <div *ngIf="questionsToCollapsed[CoursesSectionQuestions.COURSE_ADD_STUDENTS]" @collapseAnim>
-      <div class="card">
-        <div class="card-body">
-          <p>
-            There are two ways to enroll students in a course:
-          </p>
-          <ul>
-            <li>
-              From the <b>Home</b> page, click on the <button class="btn btn-primary btn-sm">Students</button> button of the corresponding course, and choose <b>Enroll</b>.
-            </li>
-            <li>
-              From the <b>Courses</b> page, click on the <button class="btn btn-secondary btn-sm">Enroll</button> button of the corresponding course.
-            </li>
-          </ul>
-        </div>
-      </div>
-      <br/>
-    </div>
-  </div>
+  <tm-instructor-help-panel class="instr-help-qn-first" #question [hidden]="key && !showQuestion.includes(CoursesSectionQuestions.COURSE_ADD_STUDENTS)"
```

Comment:
Can we remove the `#question`s? Don't think they are used anymore.

Reply:
No. That is how each panel is read and has the content indexed.